package.json

```
@@ -2,12 +2,16 @@
 	"name": "youperiod.app",
 	"version": "0.0.1",
 	"scripts": {
-		"start": "node server.js"
+		"start": "node server.js",
+		"prettify": "prettier --write ."
```

Alternatively, explicitly pass the files to format like so:
```suggestion
		"prettify": "prettier --write server.js web"
```